media-source-respec.html

```diff
@@ -824,16 +865,26 @@ <h4><dfn>Attaching to a media element</dfn></h4>
                       <li>[=Queue a task=] to [=fire an event=] named {{sourceopen}} at the <a>MediaSource</a>.</li>
                     </ol></dd>
                   </dl>
-                <li>Continue the <a def-id="resource-fetch-algorithm"></a> by running the remaining <a def-id="Otherwise-mode-is-local"></a> steps, with these clarifications:
+                <li>Continue the [=resource fetch algorithm=] by running the remaining
+                  <a def-id="Otherwise-mode-is-local"></a> steps, with these clarifications:
                   <ol>
-                    <li>Text in the <a def-id="resource-fetch-algorithm"></a> or the <a def-id="media-data-processing-steps-list"></a> that refers to "the download", "bytes received", or "whenever new data for the current media resource becomes available" refers to data passed in via {{SourceBuffer/appendBuffer()}}.</li>
-                    <li>References to HTTP in the <a def-id="resource-fetch-algorithm"></a> and the <a def-id="media-data-processing-steps-list"></a> do not apply because the HTMLMediaElement does not fetch media data via HTTP when a <a>MediaSource</a> is attached.</li>
+                    <li>Text in the [=resource fetch algorithm=] or the
+                      <a def-id="media-data-processing-steps-list"></a> that refers to "the download", "bytes received",
+                      or "whenever new data for the current media resource becomes available" refers to data passed in
+                      via {{SourceBuffer/appendBuffer()}}.</li>
+                    <li>References to HTTP in the [=resource fetch algorithm=] and the
+                      <a def-id="media-data-processing-steps-list"></a> do not apply because the HTMLMediaElement does
```

Comment:
Perhaps "shall not apply because the HTMLMediaElement shall not" ?

Reply:
Sounds good to me. Done in 16d5b78